addons/kodi.guilib/addon.xml.in

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="kodi.guilib" version="@guilib_version@" provider-name="Team-Kodi">
+  <backwards-compatibility abi="@guilib_version@"/>
```

Comment:
shouldn't these be split between  KODI_GUILIB_API_VERSION and KODI_GUILIB_MIN_API_VERSION ?
i think the generated addon.xml also needs to be added to .gitignore